src/context/runTest.js

```diff
@@ -4,8 +4,10 @@ const headers = { 'Content-Type': 'application/json' }
 
 export default function runTest(snapshot, update) {
   const {
-    host = 'localhost',
-    port = 3000,
+    serverInfo: {
+      host = 'localhost',
```

Comment:
Oh yes, using jest globals makes a lot of sense. It's my preferred solution if it works. Thanks!